content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,74 +1,355 @@
+
+   Click :guilabel:`Validate` and a pop-up window appears with a :guilabel:`Validated Address` and
+   :guilabel:`Original Address` listed. If the :guilabel:`Validated Address` is the correct mailing
+   address for tax purposes click :guilabel:`Save Validated`.
 
-.. _avatax/tax-calculation:
+   .. image:: avatax/validate-address.png
+      :align: center
+      :alt: Validate address pop-up window in Odoo with save validated button and validated address
+            highlighted.
 
-Tax calculation
+.. warning::
+   All previously entered addresses for contacts in the Odoo database will need to be validated
+   using the manually validate process outlined above. Addresses are not automatically validated if
+   they were entered previously. This only occurs upon tax calculation.
+
+Sync parameters
+---------------
+
+Upon finishing the configuration and settings of the AvaTax section click the :guilabel:`Sync
+Parameters`. This action will synchronize the exemption codes from AvaTax.
+
+.. _avatax/fiscal_positions:
+
+Fiscal position
 ===============
 
-Automatically calculate taxes on Odoo quotations and invoices with Avatax by confirming the
-documents. Alternatively, calculate the taxes manually by clicking the :guilabel:`Compute taxes
-using Avatax` button while these documents are in draft mode.
+Next, navigate to :menuselection:`Configuration --> Accounting --> Fiscal Positions`. A new
+:guilabel:`Fiscal Position` is listed called :guilabel:`Automatic Tax Mapping (AvaTax)`. Click on
+this :guilabel:`Fiscal Position`.
 
-Use the :guilabel:`Avalara Code` field that's available on customers, quotations, and invoices to
-cross-reference data in Odoo and Avatax.
+Ensure that the :guilabel:`Use AvaTax API` checkbox is ticked.
 
-.. seealso::
-   - :doc:`fiscal_positions`
+Optionally, tick the checkbox next to the field labeled: :guilabel:`Detect Automatically`. Should
+this option be ticked, then, Odoo will automatically apply this :guilabel:`Fiscal Position` for
+transactions in Odoo.
+
+Specific parameters, such as :guilabel:`VAT required`, :guilabel:`Foreign Tax ID`,
+:guilabel:`Country Group`, :guilabel:`Country`, :guilabel:`Federal States`, or :guilabel:`Zip Range`
+can be specified below this checkbox.
+
+.. warning::
+   Should the :guilabel:`Detect Automatically` checkbox not be ticked, then each *Customer* will
+   need to have the :guilabel:`Fiscal Position` set on their :guilabel:`Sales and Purchase` tab of
+   the contact record. To do so, navigate to :menuselection:`Sales app --> Order --> Customers` or
+   :menuselection:`Contacts app --> Contacts`. Then select and customer or contact to set the fiscal
+   position on.
+
+   Navigate to the :guilabel:`Sales and Purchase` tab and down to the section labeled
+   :guilabel:`Fiscal Position`. Set the :guilabel:`Fiscal Position` field to the fiscal position
+   for the customer.
+
+   .. seealso::
+      - :doc:`fiscal_positions`
```

Comment:
only need a bullet point when there's more than one link in the seealso admonition (:
```suggestion
      :doc:`fiscal_positions`
```